lmp/bb-config.sh

```diff
@@ -47,7 +47,7 @@ OSTREE_BRANCHNAME = "${MACHINE}-${OSTREE_BRANCHNAME}"
 GARAGE_SIGN_REPO = "/tmp/garage_sign_repo"
 GARAGE_TARGET_VERSION = "${H_BUILD}"
 GARAGE_TARGET_URL = "https://ci.foundries.io/projects/${H_PROJECT}/builds/${H_BUILD}"
-GARAGE_CUSTOMIZE_TARGET = "${HERE}/customize-target.sh ${GARAGE_CUSTOMIZE_TARGET_PARAMS}"
+GARAGE_CUSTOMIZE_TARGET = "${HERE}/customize-target.sh ${FACTORY} ${OTA_LITE_TAG} ${GARAGE_CUSTOMIZE_TARGET_PARAMS}"
```

Comment:
It could be also added at line #21.

Reply:
I saw that but we actually can't because that line is given in a way that it will keep the ${} format so it can later be used by bitbake, and these 2 variables are not exported to bitbake.

Which is why I added here, is it could be translated from shell to local.conf directly on execution.

Reply:
:+1:

Reply:
FACTORY -> LMP_DEVICE_FACTORY
for the tag we can set OTA_LITE_TAG = "${OTA_LITE_TAG}" in local.conf.

`export` is really required for those variables that are needed by python scripts called from recipes.

Anyway, this is really not important, it's even better, iirc, I did this trick in the first place because some of the variables are not defined at this stage (${IMAGE_BASENAME} ${TARGET_ARCH} at least back then).